domains/xrap1dx.json

```diff
@@ -1,20 +1,23 @@
 {
-    "description": "making a cool portfolio website.",
-    "repo": "https://github.com/xrap1dx?tab=repositories",
-    "owner": {
-        "username": "xrap1dx",
-        "email": "sirrapid155@gmail.com"
-    },
-    "record": {
-        "A": [
-            "69.30.249.53"
-        ],
-        "MX": [
-            "mx1.improvmx.com",
-            "mx2.improvmx.com"
-        ],
-        "TXT": [
-            "v=spf1 include:spf.improvmx.com ~all"
-        ]
+  "description": "making a cool portfolio website.",
+  "repo": "https://github.com/xrap1dx?tab=repositories",
+  "owner": {
+    "username": "xrap1dx",
+    "email": "sirrapid155@gmail.com"
+  },
+  "record": {
+    "A": [
+      "69.30.249.53"
+    ],
+    "MX": [
+      "mx1.improvmx.com",
+      "mx2.improvmx.com"
+    ],
+    "TXT": [
+      "v=spf1 include:spf.improvmx.com ~all"
+    ],
+    "CNAME": {
+      "status": "statuspage.betteruptime.com"
     }
+  }
```

Comment:
You can't have CNAME with other records
either convert it to an A record or remove it